packages/plugins/Traderxyz/src/Worker/index.ts

```diff
@@ -0,0 +1,9 @@
+import type { Plugin } from '@masknet/plugin-infra'
+import { base } from '../base'
+
+const worker: Plugin.Worker.Definition = {
```

Comment:
Looks like you're not using the Worker entry, please remove this file and the entry.

Reply:
Done.